seismiqb/src/crop_batch.py

```diff
@@ -289,7 +289,7 @@ def _correct_point_to_grid(self, point, shape, grid_src='quality_grid', eps=3):
 
     @action
     @inbatch_parallel(init='indices', post='_assemble', target='for')
-    def load_cubes(self, ix, dst, src='locations', **kwargs):
+    def load_cubes(self, ix, dst, src='locations', src_geometry='geometries', **kwargs):
```

Comment:
```suggestion
    def load_cubes(self, ix, dst, src_locations='locations', src_data='geometries', **kwargs):
```

I suggest using `src_data` instead of `src_geometry`, since for example in case of estuaries `load_crop()` is called upon an instance of `RectifiedAmplitudes`, which is definitely not a geometry in the usual sense.

And replacing `src` with `src_locations` makes arguments' names more consistent.